abstract.js

```diff
@@ -31,7 +31,15 @@ function abstractPersistence (opts) {
 
     _persistence(function (err, instance) {
       if (instance) {
+        if (instance.on) {
```

Comment:
maybe we can accept an option for this, as a persistence might use a different event, or be an eventemitter for other reasons.